torch/csrc/jit_decomp_interface.cpp

```diff
@@ -0,0 +1,21 @@
+#include <torch/csrc/jit_decomp_interface.h>
+
+namespace torch {
+namespace autograd {
+namespace impl {
+
+namespace {
+JitDecompInterface* impl = nullptr;
+}
+
+void setJitDecompImpl(JitDecompInterface* impl_) {
+  impl = impl_;
+}
+
+JitDecompInterface* getJitDecompImpl(c10::string_view name) {
+  return impl;
+}
+
```

Comment:
Based on the namespace I would have expected this file to go under torch/csrc/autograd